lib/packer/sinatra.rb

```diff
@@ -9,6 +9,7 @@ module Sinatra
     def self.registered(app)
       Packer.instance = Packer::Instance.new(
         root_path: app.settings.root,
+        config_path: app.settings.respond_to?(:packer_config_path) ? app.settings.packer_config_path : nil,
```

Comment:
I think this can be reduced to: `config_path: app.settings&.packer_config_path,`

Reply:
@budmc29 I tried that but `&` doesn't work if the object doesn't respond to the method being checked. `:try()` would work but I'd prefer not to rely on ActiveSupport methods.

Reply:
That makes sense 👍